### DISCUSSION TITLE
Ignore double clicks and conditional control tooltips

### DISCUSSION THREAD

**Discussion by WhoMI7** [tomasklaen/uosc#427] [Q&A]
1. Is it possible to make double left click on uosc elements (topbar, timeline, buttons, volume bar, etc) _not_ cycle full-screen?

2. I added a Pause/Play button, is it possible for the display name to be `Play` when video is paused and `Pause` when video is playing?

**Comment by tomasklaen** [comment]
### 1.

We are already trying to ignore double clicks at:
    https://github.com/tomasklaen/uosc/blob/5a02c6d205fb0993adf07aacb894b4d444ff34b1/scripts/uosc_shared/elements/Elements.lua#L159
    But it doesn't look like it's working :/ Not sure what else can be done here.

### 2.

What do you mean by display name? Tooltip? Than no. Just make it `Play/Pause`. Asking for the already convoluted controls config syntax to also support conditional tooltips is too much...

**Reply by hooke007** [reply]
> But it doesn't look like it's working

I noticed that there might be some "action" was inserted into two clicks?
If you tried to move the cursor to the volunme bar, and then quickily click twice. You would see that the value of volume was always tweaked to 0.

**Reply by hooke007** [reply]
@christoph-heinrich It seems to be related? https://github.com/mpv-player/mpv/issues/11154#issuecomment-1385756459

**Reply by christoph-heinrich** [reply]
Yes that's certainly the underlying problem. Copying the workaround from there to the beginning of `uosc.lua` makes it work.
However I'm not in favor of that because
1. binding to `mouse_move` can cause compatiblity problems #295
2. that problem should really be solved in mpv itself
